### PR TITLE
Docker Swarm

### PR DESCRIPTION
**Pull request by pallenSL**
fix the docker cluster address (eth1)
add a srm-swarm-agent role and add to srm-nodes playbook so they all advertise
add a srm-swarm-master role and use in new srm-master playbook
update README

### CODE REVIEW

File: dataCenter/ansible/roles/srm-swarm-master/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+- name: pull
+  docker_image:
+    name: swarm
+
+- name: start
+  command: docker run -d -p 3376:2375 swarm manage consul://{{ consul_address }}
```
Comment (jebbinSCL) [inline dataCenter/ansible/roles/srm-swarm-master/tasks/main.yml]: Why are these ports different?

Comment (pallenSL) [inline dataCenter/ansible/roles/srm-swarm-master/tasks/main.yml]: We need the docker daemon and the swarm master running.  By default they would fight over 2375.  I think I could use 
docker run -d -p 3376:3376 swarm manage -H tcp://{{ eth1 }}:3376 consul://{{ blah }}
if that would be clearer.

Comment (jebbinSCL) [inline dataCenter/ansible/roles/srm-swarm-master/tasks/main.yml]: Its not to much of an issue, but probably worth a note for future reference. Could you either go with that change, or make a comment to explain it for whoever sees it in the future?

Comment (pallenSL) [inline dataCenter/ansible/roles/srm-swarm-master/tasks/main.yml]: Will change it, test it, commit again.

**Comment by jebbinSCL** [comment]
:shipit:

**Pull request merged**